README.md

```
@@ -24,8 +25,65 @@ and displays information about the current train and next station.
+-p, --prune     Prune existing Docker volumes to reset persisted data
+-v, --version   Print the current version of this script
+```
+
+_You can also do this manually by running `docker compose up`._
+
+### [`stop.sh`](stop.sh)
+
+Use the [`stop.sh`](stop.sh) script to stop running Docker images.
+
+```bash
+$ ./stop.sh [<flags>]
```

markdownlint-cli2 (0.20.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)